test/functional/configuration/configuration.spec.js

```diff
@@ -140,6 +141,7 @@ test("initializes form fields with full settings", async () => {
   await instances[1].specificContext.deviceField.expectUnchecked();
   await instances[1].specificContext.environmentField.expectUnchecked();
   await instances[1].specificContext.placeContextField.expectUnchecked();
+  await instances[1].specificContext.highEntropyUserAgentHintsContextField.expectUnchecked();
 
   await instancesTabs.selectTab("alloy3");
 
```

Comment:
Add or augment test cases to make sure that:
 * checking the new checkbox leads to the correct context array in settings.
 * loading the settings with the new context automatically checks the new checkbox.